Resources/Prototypes/Entities/Objects/Tools/crowbars.yml

```diff
@@ -0,0 +1,195 @@
+- type: entity
+  name: crowbar
+  parent: BaseItem
+  id: BaseCrowbar
+  abstract: true
+  components:
+  - type: EmitSoundOnLand
+    sound:
+      path: /Audio/Items/crowbar_drop.ogg
+  - type: Tag
+    tags:
+    - Crowbar
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    damage:
+      types:
+        Blunt: 8
+    soundHit:
+      collection: MetalThud
+  - type: Tool
+    qualities:
+      - Prying
+    useSound:
+      path: /Audio/Items/crowbar.ogg
+  - type: ToolTileCompatible
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 100
+  - type: StaticPrice
+    price: 22
+  - type: Prying
+  - type: Clothing
+    quickEquip: false
+    slots:
+    - Belt
+  - type: Item
+    size: Normal
+    shape:
+    - 0,0,0,1
+
+# Standard (grey) Crowbar
+- type: entity
+  parent: BaseCrowbar
+  id: Crowbar
+  description: A multipurpose tool to pry open doors and fight interdimensional invaders.
+  components:
+  - type: Sprite
+    sprite: Objects/Tools/crowbar.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Tools/crowbar.rsi
+    storedSprite:
+      sprite: Objects/Tools/crowbar.rsi
+      state: storage
+
+# Emergency (red) Crowbar
+- type: entity
+  name: emergency crowbar
+  parent: BaseCrowbar
+  id: CrowbarRed
+  description: A pocket crowbar designed to pry open doors and firelocks during power outages.
+  components:
+  - type: Tag
+    tags:
+    - Crowbar
+    - CrowbarRed
+  - type: Sprite
+    sprite: Objects/Tools/crowbar.rsi
+    layers:
+    - state: icon
+    - state: cover-icon
+      color: "#990000"
+  - type: Item
+    sprite: Objects/Tools/crowbar.rsi
+    storedSprite:
+      state: red-storage
+      sprite: Objects/Tools/crowbar.rsi
+    inhandVisuals:
+      left:
+      - state: inhand-left
+      - state: cover-inhand-left
+        color: "#990000"
+      right:
+      - state: inhand-right
+      - state: cover-inhand-right
+        color: "#990000"
```

Comment:
you should write `size: Small` for it to fit in pockets (as said in desc)

Reply:
i thought they said it was okay